main.go

```diff
@@ -169,10 +165,8 @@ func main() {
             </html>`))
 	})
 
-	level.Info(logger).Log("msg", "Listening on", "address", *listenAddress)
```

Comment:
Thanks! Can we keep logging the listening addresses, though? E.g. one log line per address?

Reply:
Logging is now handled by the toolkit. No need for it here anymore since we can have multiple listen addresses.